webcomponents/tool/src/main/frontend/js/grader/sakai-grader.js

```diff
@@ -585,45 +598,101 @@ export class SakaiGrader extends gradableDataMixin(SakaiElement) {
     if (!feedbackPanel.dialog("instance")) {
       feedbackPanel.dialog({
         width: "auto",
-        close: () => this.toggleFeedback()
+        beforeClose: () => { return this.cancelFeedbackToggle() },
```

Comment:
You don't need the return here. () => this.cancelFeedbackToggle(),

Reply:
I think that I need to prevent close before confirm if we want to close. I don't know if there is a better approach.